api/server/router/volume/volume.go

```diff
@@ -44,6 +44,14 @@ func (r *router) initRoutes() {
 			handlers.NewSchemaValidator(nil, schema.ServiceVolumeMapSchema, nil),
 		),
 
+		// get all volumes from all services with attachments
+		httputils.NewGetRoute(
```

Comment:
I'm not sure why this was added. It's unnecessary as the actual handler checks for the query string. Additionally, the same name cannot be used for two routes or one will overwrite the other.